wikiconv/conversation_reconstruction/construct_utils/conversation_constructor.py

```diff
@@ -430,7 +430,7 @@ def clean_dict(self, page, the_dict):
       The resulting dictionary.
     """
     keylist = the_dict.keys()
-    ret = the_dict
+    ret = the_dict.copy()
```

Comment:
is this necessary for py3, or is it just a code improvement that you noticed? (no need to revert if the latter, i don't mind having pull requests not be laser focused on one thing, just curious.)

Reply:
This is in fact required due to a check that only surfaces in python3. It was wrong.